docs/CHANGELOG.asciidoc

```diff
@@ -89,6 +90,25 @@ This section summarizes the changes in each release.
 //=== Known Issues
 ////
 
+[[release-notes-5.6.15]]
+== {kib} 5.6.15
+coming[5.6.15]
+
+[float]
+[[Enhancements-5.6.15]]
+=== Enhancements
+
+Operations::
+* Bumps node to 6.16.0 {pull}27921[#27921]
+
+[float]
+[[bug-fixes-5.6.15]]
+=== Bug Fixes
+
+Timelion::
+
```

Comment:
I don't think it's a big deal, but there's an extra space under the Timelion heading